### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by rvrvrv**
Minor punctuation and capitalization edits

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,19 +20,19 @@ Choose an overview of your container infrastructure, or focus on a specific micr
...
 
-Interact with your containers directly: pause, restart and stop containers. Launch a command line. All without leaving the scope browser window.
+Interact with your containers directly: pause, restart, and stop containers. Launch a command line. All without leaving the scope browser window.
 
 ### Extend and customize via plugins
 
-Add custom details or interactions for your hosts, containers and/or processes by creating Scope plugins; or just choose from some that others have already written at the Github [Weaveworks Scope Plugins](https://github.com/weaveworks-plugins/) organization.
+Add custom details or interactions for your hosts, containers, and/or processes by creating Scope plugins. Or, just choose from some that others have already written at the GitHub [Weaveworks Scope Plugins](https://github.com/weaveworks-plugins/) organization.
 
-## <a name="getting-started"></a>Getting started
```
Comment (dholbach) [inline README.md]: Not a strong opinion, but I would leave this "title case".

File: README.md [hunk 2]
```diff
@@ -44,18 +44,18 @@ This script downloads and runs a recent Scope image from Docker Hub.
 Now, open your web browser to **http://localhost:4040**. (If you're using
 boot2docker, replace localhost with the output of `boot2docker ip`.)
 
-For instructions on installing Scope on [Kubernetes](https://www.weave.works/docs/scope/latest/installing/#k8s), [DCOS](https://www.weave.works/docs/scope/latest/installing/#dcos) or [ECS](https://www.weave.works/docs/scope/latest/installing/#ecs), see [the docs](https://www.weave.works/docs/scope/latest/introducing/).
+For instructions on installing Scope on [Kubernetes](https://www.weave.works/docs/scope/latest/installing/#k8s), [DCOS](https://www.weave.works/docs/scope/latest/installing/#dcos), or [ECS](https://www.weave.works/docs/scope/latest/installing/#ecs), see [the docs](https://www.weave.works/docs/scope/latest/introducing/).
 
-## <a name="help"></a>Getting help
```
Comment (dholbach) [inline README.md]: Same as above.

File: README.md [hunk 3]
```diff
@@ -20,19 +20,19 @@ Choose an overview of your container infrastructure, or focus on a specific micr
...
 
 <img src="imgs/terminals.png" width="200" alt="Launch a command line." align="right">
 
-Interact with your containers directly: pause, restart and stop containers. Launch a command line. All without leaving the scope browser window.
+Interact with your containers directly: pause, restart, and stop containers. Launch a command line. All without leaving the scope browser window.
 
 ### Extend and customize via plugins
 
-Add custom details or interactions for your hosts, containers and/or processes by creating Scope plugins; or just choose from some that others have already written at the Github [Weaveworks Scope Plugins](https://github.com/weaveworks-plugins/) organization.
+Add custom details or interactions for your hosts, containers, and/or processes by creating Scope plugins. Or, just choose from some that others have already written at the GitHub [Weaveworks Scope Plugins](https://github.com/weaveworks-plugins/) organization.
```
Comment (dholbach) [inline README.md]: Not a native speaker here, but is "Or, ..." a good way to start a sentence, or should we make it two entirely separate sentences?

File: README.md (hunk 1 above)
Comment (rvrvrv) [reply README.md]: "Getting Started" would be title case (in English).

File: README.md (hunk 2 above)
Comment (rvrvrv) [reply README.md]: "Getting Help" would be title case (in English).

File: README.md (hunk 3 above)
Comment (rvrvrv) [reply README.md]: It is acceptable to start a sentence with "Or." In this context, I think it divides the two options a little more clearly than one long sentence. Either way works :+1:

File: README.md (hunk 2 above)
Comment (dholbach) [reply README.md]: Hah, yes! :-)

File: README.md (hunk 1 above)
Comment (dholbach) [reply README.md]: Entirely right, yes!

File: README.md (hunk 3 above)
Comment (dholbach) [reply README.md]: Ok, sounds good.

**Pull request merged**